plaso/cli/extraction_tool.py

```diff
@@ -103,16 +107,21 @@ def _CreateProcessingConfiguration(self, knowledge_base):
           'operating_system_product')
       operating_system_version = knowledge_base.GetValue(
           'operating_system_version')
-      parser_filter_expression = (
-          parsers_manager.ParsersManager.GetPresetForOperatingSystem(
+      preset_definitions = (
+          parsers_manager.ParsersManager.GetPresetsForOperatingSystem(
               operating_system, operating_system_product,
               operating_system_version))
 
-      if parser_filter_expression:
-        logger.info('Parser filter expression changed to: {0:s}'.format(
-            parser_filter_expression))
+      if preset_definitions:
+        if len(preset_definitions) != 1:
+          raise errors.BadConfigOption(
+              'More than 1 parser presets found for: {0:s}'.format(
```

Comment:
"more than one parser preset", not "parser presets". Also, please print the preset names here to aid debugging.

Reply:
ack